app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :users
+  belongs_to :posts
+
+  def update_comment_counter
+    posts.comments_counter = comment.count
+  end
```

Comment:
- [ ] The same here, Great job creating a method that updates the comments counter for a post. 👏🏻 👏🏻  However It would be nice to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) -* `after_save` to call the `#comments_posts_counter` method every time a new like has been saved to the database, then make the `comments_posts_counter` method to be a `private` method 👍🏻 💯 
- 🤔  Your code should look like this 👇🏻 :

```ruby
# app/models/comment.rb
...
     after_save : comments_posts_counter
   
     private

    def comments_posts_counter
       post.increment!(:likes_counter)
    end
```